packages/cra-template-edu/template/src/components/Navigation/Navigation.test.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import {generateSnapshots} from '@chanzuckerberg/story-utils';
+import {composeStories} from '@storybook/testing-react';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import * as stories from './Navigation.stories';
+
+const {Home} = composeStories(stories);
+
+generateSnapshots(stories);
+
+it('bolds the active item', async () => {
```

Comment:
Interestingly, running the tests for this repo (frontend-libs) still runs these tests from the template.

Didn't exactly expect that, but it makes sense, and seems useful to make sure they'll actually pass in generated apps.